sphinx/writers/latex.py

```diff
@@ -1376,6 +1376,9 @@ def visit_table(self, node):
         self.table = Table(node)
         if self.next_table_colspec:
             self.table.colspec = '{%s}\n' % self.next_table_colspec
+            if 'colwidths-given' in node.get('classes', []):
+                logger.info('both tabularcolumns and :widths: option are given. '
+                            ':widths: is ignored.', location=node)
```

Comment:
are you planning to make this translatable string?

Reply:
oh I'd forgotten that.

Anyway, Sphinx only translates strings on documents, so these messages are not translated.
`__(...)` (double underscore) helper is provided, but it is not implemented now.
https://github.com/sphinx-doc/sphinx/blob/122a404eb3cf0c50f5ada60fc236d8b460c32492/sphinx/locale/__init__.py#L251-L258